avocado/utils/base_utils.py

```diff
@@ -0,0 +1,61 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Author: Santhosh G <santhog4@linux.vnet.ibm.com>
+#
+# This code was inspired in the autotest project,
+# client/shared/base_utils.py
+
+import os
+import re
+
+
+def check_kernel_ver(ver):
+    kernel_ver = os.uname()[2]
+    kv_tmp = re.split(r'[-]', kernel_ver)[0:3]
+    # In compare_versions, if v1 < v2, return value == -1
+    if compare_versions(kv_tmp[0], ver) == -1:
+        raise Exception("Kernel too old (%s). Kernel > %s is needed." %
```

Comment:
Btw it's not really nice to raise pure Exceptions. I'd go with AssertionError, RuntimeError or ValueError (ideally using the `assert foo >= bar, "Foo too old"`.